physionet-django/console/templatetags/console_templatetags.py

```diff
@@ -38,3 +38,11 @@ def smooth_timedelta(timedelta):
     seconds -= minutes * 60
 
     return f'{int(hours):02d}:{int(minutes):02d}:{int(seconds):02d}'
+
+
+@register.filter(name='yesno')
+def yesno(is_credentialed, options):
```

Comment:
What is that? Is that just `yesno` reimplemented? Django already includes this functionality.